### PR TITLE
Correct calculation error with curve friction

### PR DESCRIPTION
**Pull request by peternewell**
Review and adjust curve friction calculations.

https://bugs.launchpad.net/or/+bug/2030936

### CODE REVIEW

File: Source/Orts.Simulation/Simulation/RollingStocks/TrainCar.cs [hunk 1]
```diff
@@ -819,6 +821,81 @@ public virtual void Initialize()
...
+                        {
+                            RigidWheelBaseM = 3.5052f;       // Assume a standard 6 wheel bogie (3 axle) locomotive - wheel base - 11' 6" (3.5052m)
+                        }
+                    }
+                    else // assume steam locomotive
+                    {
+
+                        if (LocoNumDrvAxles >= Axles) // Test to see if ENG file value is too big (typically doubled)
+                        {
+                            LocoNumDrvAxles = LocoNumDrvAxles / 2;  // Appears this might be the number of wheels rather then the axles.
```
Comment (cjakeman) [inline Source/Orts.Simulation/Simulation/RollingStocks/TrainCar.cs:885]: Agree with the aim here, but would be helpful to write a message to the logfile.

**Review by cjakeman — DISMISSED** [review]
Hi Peter,

This is great. All logic clearly explained.

Approved, with one comment. I'll let you merge when you're ready.

Chris

File: Source/Orts.Simulation/Simulation/RollingStocks/TrainCar.cs (hunk 1 above)
Comment (peternewell) [reply Source/Orts.Simulation/Simulation/RollingStocks/TrainCar.cs:901]: Message added.

**Comment by peternewell** [comment]
PR closed by mistake.

**Review by cjakeman — APPROVED** [review]
Hi Peter,

Thanks for making those changes,

Approved.

Chris

**Comment by peternewell** [comment]
Chris,

Thanks for that.

**Pull request merged**